tasks/main.yml

```diff
@@ -83,6 +83,7 @@
     from_port: "{{ item.from_port | default('') }}"
     to_port: "{{ item.to_port | default('') }}"
     protocol: "{{ item.protocol | default('any') }}"
+    route: "{{ item.route | default('') }}"
```

Comment:
Is this backwards compatible?

Reply:
It should be, https://docs.ansible.com/ansible/latest/modules/ufw_module.html mentions nothing about the `route` parameter being added in a specific version of ansible which makes me assume that this parameter would have been added when the ufw module was first added in ansible 1.6.